packages/components/src/components/Tabs/hooks/useTabTriggers.tsx

```diff
@@ -0,0 +1,140 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { cloneElement, type RefObject, useEffect, useLayoutEffect, useRef, useState } from 'react';
+
+import { type TabTrigger } from '../types';
+
+const getOverflowingTriggers = (triggers: TabTrigger[], triggerListElement: HTMLDivElement) => {
+    const triggersOutOfView: TabTrigger[] = [];
+    if (triggerListElement) {
```

Comment:
to avoid the pyramid we could do 
```suggestion
    if (!triggerListElement) {
      return triggersOutOfView;
    }
```

Reply:
fixed